docs/charts.md

```diff
@@ -763,14 +764,72 @@ parent chart.
 
 Also, global variables of parent charts take precedence over the global variables from subcharts.
 
+### Schema Files
+
+Sometimes, a chart writer might want to define a structure on their values.
```

Comment:
can you change "chart writer" to "chart maintainer" that way it's consistent with the rest of the docs.

Reply:
done